{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--fa2faf8b-ea7b-4bc5-9cac-9d1789f22ce3",
    "created": "2020-05-26T20:29:00.712895Z",
    "modified": "2020-05-26T20:29:00.712895Z",
    "name": "CVE-2013-4189",
    "description": "Multiple unspecified vulnerabilities in (1) dataitems.py, (2) get.py, and (3) traverseName.py in Plone 2.1 through 4.1, 4.2.x through 4.2.5, and 4.3.x through 4.3.1 allow remote authenticated users with administrator access to a subtree to access nodes above the subtree via unknown vectors."
}